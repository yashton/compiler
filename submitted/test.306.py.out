(ID "foo")
(ID "bar")
(ERROR "Illegal character '\ 
   baz ' on line 1)
