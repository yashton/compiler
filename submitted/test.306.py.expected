(ID "foo")
(ID "bar")
(ERROR "unknown char: '\'")
(NEWLINE)
(INDENT)
(ID "baz")
(DEDENT)
(ENDMARKER)
